gym/f110_gym/envs/f110_env.py

```diff
@@ -255,6 +241,14 @@ def configure(self, config: dict) -> None:
             if hasattr(self, "sim"):
                 self.sim.update_params(self.config["params"])
 
+            if hasattr(self, "action_space"):
+                # if some parameters changed, recompute action space
+                action_type_fn = CarActionEnum.from_string(self.config["control_input"])
+                self.action_type = action_type_fn(params=self.params)
```

Comment:
Should be using `self.config["params"]` here? I don't think `self.pararms` is updated after init.